go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/nspcc-dev/locode-db
 
-go 1.22
+go 1.23
```

Comment:
If there are no code changes you can leave it as is, see hrw and many other libs we already updated.